### DISCUSSION TITLE
Ignore me ;)

### DISCUSSION THREAD

**Discussion by bettywolfy** [carbon-language/carbon-lang#2369] [General]

**Comment by bettywolfy** [comment]
I not good at this...

**Comment by jonmeow** [comment]
Per the FAQ, [Why isn't there a Carbon Language logo?](https://github.com/carbon-language/carbon-lang/blob/trunk/docs/project/faq.md#why-isnt-there-a-carbon-language-logo):

> Establishing a Carbon Language logo isn't a priority right now. Remember that this project is an experiment, and so we think it's best to concentrate efforts on ensuring that the language succeeds at its goals instead.
> 
> We have a few drafts in the works, but it requires a fair amount of work to get right, and getting it wrong is costly, so we won't be adding one in the near future. Don't suggest logos, because we need to be careful about how we create one.

This is something we need to approach carefully, and I've removed the image because of the issues involved.

**Reply by bettywolfy** [reply]
Can you close this? 😅
